20240329_CHWIDE_component_separated/README.md

```diff
@@ -0,0 +1,28 @@
+# Documentation for CHWIDE component separated maps
+
+#### Data description
+We provide foreground cleaned CMB maps obtained with NILC for CMB-S4 Chile wide survey. There are separate maps for E and B-mode polarizations. 
+Currently, about 25 realizations are provided for testing purposes. The masks used for the E and B maps are also provided. All maps are at `NSIDE=2048` 
+and `lmax=4096`, with 2.1 arcmin gaussian beam. The current maps are only with the PySM medium complexity foreground model.
+
+#### Data availability
+They on the CMB-S4xLiteBIRD shared space on NERSC: `/global/cfs/cdirs/cmbs4xlb/v1/component_separated/chwide`
+The E-maps are in `nilc_Emaps/fits` subdir, the B-maps are in `nilc_Bmaps/fits` subdir, the common masks are in `masks_common`.
+
+#### Mask description
+There are 3 masks:
+
+1. `chwide_clip0p3relhits_NSIDE2048.fits` &rarr; footprint of the CHWIDE patch after clipping to remove high noise regions.
+2. `chwide_clip0p3relhits_3degC2apo_NSIDE2048.fits` &rarr; The above region with 3deg apodization. All E-mode maps use this mask. There is no Galactic mask for the NILC E-maps.
+3. `dust-mask-15pc_3pc-apo_NSIDE2048.fits` &rarr; The dust mask masks brightest 15 percent of the sky in polarised dust emission, and an additional 3 percent apodization. This mask is used for the NILC B-maps, in addition to the apodized footprint mask.
+
+For E-maps use mask 2, while for B-maps use mask 2 x mask 3.
```

Comment:
`x` means union I guess?

Reply:
Yes, and I added an additional clarification for this.